lib/__tests__/util.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { safeToString } from '../utils'
+
+describe('safeToString', () => {
+  const basic = [
+    undefined,
+    null,
+    true,
+    'string',
+    123,
+    321n,
+    { object: 'yes' },
+    [1, 'hello', true, null],
+    (a: number, b: number) => a + b,
+    Symbol('safeToString'),
+  ]
+  const testCases = [
+    ...basic.map((input) => [input, String(input)]),
```

Comment:
Instead of mapping the `basic` values and relying on `String(input)` to build the `[input, output]` pair, maybe this should explicitly declare the values.  E.g.;

```js
const testCases = [
  [undefined, 'undefined'],
  [null, 'null'],
  [true, 'true'],
  ['string', 'string'],
  ...
  [
    [Object.create(null), Symbol('safeToString')],
    '[[object Object], Symbol(safeToString)]',
  ],
]
```